### PR TITLE
Easeware.DriverEasy version 5.6.15

### PR DESCRIPTION
**Pull request by BaterWucket**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16164)

### CODE REVIEW

File: manifests/e/Easeware/DriverEasy/5.6.15.34863/Easeware.DriverEasy.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Easeware.DriverEasy
+PackageVersion: 5.6.15.34863
```
Comment (OfficialEsco) [inline manifests/e/Easeware/DriverEasy/5.6.15.34863/Easeware.DriverEasy.installer.yaml:4]:
```suggestion
PackageVersion: 5.6.15
```

File: manifests/e/Easeware/DriverEasy/5.6.15.34863/Easeware.DriverEasy.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Easeware.DriverEasy
+PackageVersion: 5.6.15.34863
```
Comment (OfficialEsco) [inline manifests/e/Easeware/DriverEasy/5.6.15.34863/Easeware.DriverEasy.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 5.6.15
```

File: manifests/e/Easeware/DriverEasy/5.6.15.34863/Easeware.DriverEasy.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Easeware.DriverEasy
+PackageVersion: 5.6.15.34863
```
Comment (OfficialEsco) [inline manifests/e/Easeware/DriverEasy/5.6.15.34863/Easeware.DriverEasy.yaml:4]:
```suggestion
PackageVersion: 5.6.15
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Please update the PackageVersion so it dosn't cause issues with `winget upgrade`.
Remember you also have to change the folder name.

**Comment by OfficialEsco** [comment]
Gotta change the folder name
`Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\e\Easeware\DriverEasy\5.6.15.34863', expected 'manifests\e\Easeware\DriverEasy\5.6.15'`

**Comment by BaterWucket** [comment]
Can I create a new pull request with everything correct?

**Comment by BaterWucket** [comment]
Because I didn't know how to edit folder name now

**Comment by OfficialEsco** [comment]
Sure, add `closes #16164` in the top post.
You should be using GitHub desktop for that kind of stuff :)

**Comment by BaterWucket** [comment]
Thanks

**Pull request closed without merging**